src/components/capture/create/index.tsx

```diff
@@ -74,21 +74,25 @@ const selectors = {
     changeSet: {
         addCapture: (state: ChangeSetState) => state.addCapture,
     },
-    schema: {
-        clearResources: (state: SchemaEditorState) => state.clearResources,
-        resources: (state: SchemaEditorState) => state.resources,
-    },
     notifications: {
         showNotification: (state: NotificationState) => state.showNotification,
     },
 };
 
+const notification: Notification = {
+    description: 'Your new capture is published and ready to be used.',
+    severity: 'success',
+    title: 'New Capture Created',
+};
```

Comment:
Never changes - can be kept out of the component. Might even end up making as "library" of predefined notifications?